debian/xenial/kubeadm/debian/rules

```diff
@@ -8,10 +8,9 @@ build:
 
 binary:
 	mkdir -p usr/bin
-	#curl  --fail -sS -L \
-	#	-o usr/bin/kubeadm \
-	#	"https://storage.googleapis.com/kubernetes-release/release/v{{ .Version }}/bin/linux/{{ .Arch }}/kubeadm"
-	touch usr/bin/kubeadm
+	curl  --fail -sS -L \
+		-o usr/bin/kubeadm \
+		"https://storage.googleapis.com/kubeadm/v{{ .Version }}/bin/kubeadm"
```

Comment:
@errordeveloper One thing you should fix here is that the path should be `kubeadm/v{{ .Version }}/bin/linux/{{ .Arch }}/kubeadm`
You should also push for all arches (amd64, arm and arm64)